contributing/editing-docs.rst

```diff
@@ -73,17 +73,15 @@ See :doc:`ci-docs`.
 Source
 ^^^^^^
 
-The Sphinx documentation is in the ``/docs/sphinx/`` folder in the code
-repository at `<https://github.com/openmicroscopy/bioformats>`_.
-
-Content for autogenerating documentation is in ``/components/autogen/src/``.
+The Sphinx documentation is decoupled from the code repository, at
+`<https://github.com/ome/bio-formats-documentation>`_.
 
 Building locally
 ^^^^^^^^^^^^^^^^
 
-The build uses Sphinx via ``ant``. Within the Sphinx folder, ``make clean
-html`` will generate the webpages provided you have both Sphinx and ant
-installed.
+The build uses Sphinx via ``Maven``. ``mvn`` will generate the webpages
+provided you have both Sphinx and Maven installed. To avoid running the
+linkchecker by default use ``mvn -DskipTests``.
```

Comment:
Please could you use `skipSphinxTests` here since it's more specific when used in bio-formats-build to only skip the linkchecking.